frontend/src/components/details/navbar.css

```diff
@@ -21,9 +21,9 @@
 .navbar b {
     float: left;
     padding: 12px;
-    color: white;
+    color: #dbdbdb;
```

Comment:
This usage of concrete colours everywhere is something that I would still like to avoid. Maybe we can do some research on how to define style colours in CSS during the next sprint. There has to be a way.

Reply:
I totally agree but during this sprint I was not able to figure it out. Lets do it in that way 😄  , when determining them in .css file , we do have the hover color for icon and text seperately .